src/test/suite/fixtures/recorded/surroundingPair/parseTreeParity/takePairDouble.yml

```diff
@@ -5,7 +5,7 @@ command:
   action: clearAndSetSelection
   targets:
     - type: primitive
-      modifier: {type: surroundingPair, delimiter: any, delimiterInclusion: excludeInterior}
+      modifier: {type: surroundingPair, delimiter: any}
```

Comment:
for these tests, please don't change the command to match the file name: it should always be the other way around.  For example, this test was recorded back when "pair" meant "bounds", but the name of the test was never updated

Reply:
Something is wrong on these tests. If we are referencing bounds why is there only one selection?